doc/file-layer.rst

```diff
@@ -35,59 +40,67 @@ Use-cases
   * type and size of named chunk in frame *i*
   * read data for named chunk in frame *i*
   * read only a portion of a chunk
+  * list chunk names in the file
 
 * writes
 
   * write data to named chunk in the current frame
-  * write a single data chunk from multiple MPI ranks
   * end frame and commit to disk
 
-These capabilities should enable a simple and rich higher level schema for storing particle and other types of
-data. The schema determine which named chunks exist in a given file and what they mean.
+These capabilities enable a simple and rich higher level schema for storing
+particle and other types of data. The schema determine which named chunks exist
+in a given file and what they mean.
 
 Non use-cases
 -------------
 
 These capabilities are use-cases that GSD does **not** support, by design.
 
-#. Modify data in the file: GSD is designed to capture simulation data, that raw data should not be modifiable.
+#. Modify data in the file: GSD is designed to capture simulation data, that raw
+   data should not be modifiable.
 #. Add chunks to frames in the middle of a file: See (1).
-#. Transparent conversion between float and double: Callers must take care of this.
-#. Transparent compression - this gets in the way of parallel I/O. Disk space is cheap.
+#. Transparent conversion between float and double: Callers must take care of
+   this.
+#. Transparent compression - this gets in the way of parallel I/O. Disk space is
+   cheap.
+
+Dependencies
+------------
+
+The file layer is implemented in C (*not C++*) with no dependencies to enable
+trivial installation and incorporation into existing projects. A single header
+and C file completely implement the entire file layer. Python based projects
+that need only read access can use :py:mod:`gsd.pygsd`, a pure python gsd reader
```

Comment:
"that need only read..."

Consistently capitalize "python" (also in the paragraphs below).

Reply:
Fixed.